### PR TITLE
reformat labels, add descriptions in tooltips

### PR DESCRIPTION
**Pull request by danwillm**

### CODE REVIEW

File: sidecar/main.cpp [hunk 1]
```diff
@@ -45,11 +45,6 @@ int GetSettings(nlohmann::json& json) {
           break;
         }
       }
-
-      if (err != vr::EVRSettingsError::VRSettingsError_None) {
```
Comment (noob247365) [inline sidecar/main.cpp:49]: Why are we deleting this?

File: src/components/ToolTip.svelte [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<script>
+    export let title = '';
```
Comment (noob247365) [inline src/components/ToolTip.svelte:2]: "title" is a strange description for what is being shown, I think "description" or "details" might be better

File: src/components/Input/CheckBox.svelte [hunk 3]
```diff
@@ -1,9 +1,25 @@
...
-    <span class="text-gray-900 font-medium text-sm">{label}</span>
-</label>
\ No newline at end of file
+<div class="flex flex-row">
+    <label class="flex items-center space-x-3 my-2">
+        <input type="checkbox" bind:checked={checked} name="checkbox"
+               class="form-tick appearance-none h-5 w-5 border border-gray-300 rounded-md checked:bg-blue-600 checked:border-transparent focus:outline-none cursor-pointer">
+        <span class="text-gray-900 font-medium text-sm">{configurationOptionStrings[label]?.title ?? prettyPrintLabel(label)}</span>
+    </label>
+    <div class="p-2">
```
Comment (noob247365) [inline src/components/Input/CheckBox.svelte:18]: Should the `#if` be on the outside of this div element?

File: src/components/Input/Text.svelte [hunk 4]
```diff
@@ -1,22 +1,40 @@
...
-        <input type="number" name="text-input" bind:value={value}
-               class="focus:ring-indigo-500 focus:border-indigo-500 flex-1 block rounded text-sm border-gray-300"
-               placeholder="{value}">
+<div class="flex flex-col flex-grow py-1">
+    <div class="flex flex-row">
+        <p class="block text-sm font-medium text-gray-700">
+            {configurationOptionStrings[label]?.title ?? prettyPrintLabel(label)}
+        </p>
+        <div class="px-1">
+            {#if configurationOptionStrings[label]?.description}
```
Comment (noob247365) [inline src/components/Input/Text.svelte:20]: Same question about `#if`

File: src/components/ToolTip.svelte [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+        on:mouseleave={mouseLeave}
+        on:mousemove={mouseMove}>
+    <slot/>
+</div>
+
+{#if isHovered}
+    <div style="top: {y}px; left: {x}px;" class="tooltip">{@html title}</div>
+{/if}
+
+<style>
```
Comment (noob247365) [inline src/components/ToolTip.svelte:34]: Style tags usually come before the content they're affecting. Feel free to resolve this comment without making changes

**Review by noob247365 — CHANGES_REQUESTED** [review]
Functionally it looks good, just a few minor suggestions

File: sidecar/main.cpp (hunk 1 above)
Comment (danwillm) [reply sidecar/main.cpp:49]: I think it's unnecessary, if we fail then the value is just going to be empty, which can be spotted and fixed in the ui.

File: src/components/Input/CheckBox.svelte (hunk 3 above)
Comment (danwillm) [reply src/components/Input/CheckBox.svelte:18]: good spot. resolved in later commit

File: src/components/Input/Text.svelte (hunk 4 above)
Comment (danwillm) [reply src/components/Input/Text.svelte:20]: as above

File: src/components/ToolTip.svelte (hunk 5 above)
Comment (danwillm) [reply src/components/ToolTip.svelte:34]: It's common for style tags in svelte to appear at the bottom of the file. These style tags are scoped to the component

File: src/components/ToolTip.svelte (hunk 2 above)
Comment (danwillm) [reply src/components/ToolTip.svelte]: I'll be more generic, something like `text`?

File: sidecar/main.cpp (hunk 1 above)
Comment (noob247365) [reply sidecar/main.cpp:49]: Fair enough

File: src/components/ToolTip.svelte (hunk 5 above)
Comment (noob247365) [reply src/components/ToolTip.svelte:34]: Ah, very cool. Carry on

**Review by noob247365 — APPROVED** [review]

**Pull request merged**